### PR TITLE
Screening

### PR DESCRIPTION
**Pull request by hhodzhov**

### CODE REVIEW

File: cinema-soft/src/main/java/softServe/academy/cinemasoft/controller/ScreeningController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import softServe.academy.cinemasoft.service.ScreeningService;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```
Comment (hhodzhov) [inline cinema-soft/src/main/java/softServe/academy/cinemasoft/controller/ScreeningController.java]: just @Controller

File: cinema-soft/src/main/java/softServe/academy/cinemasoft/model/Screening.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import javax.persistence.*;
+
+@Entity
+@Table(name = "screening")
+public class Screening {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer Id;
+    @Column(unique = true)
```
Comment (hhodzhov) [inline cinema-soft/src/main/java/softServe/academy/cinemasoft/model/Screening.java]: remove unique

File: cinema-soft/src/main/java/softServe/academy/cinemasoft/model/Screening.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    String startTime;
```
Comment (hhodzhov) [inline cinema-soft/src/main/java/softServe/academy/cinemasoft/model/Screening.java]: access modifier?

File: cinema-soft/src/main/java/softServe/academy/cinemasoft/service/ScreeningService.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class ScreeningService {
+
+    @Autowired
+    private ScreeningRepository screeningRepository;
+
```
Comment (hhodzhov) [inline cinema-soft/src/main/java/softServe/academy/cinemasoft/service/ScreeningService.java:15]: @Autowired should be above constructor

File: cinema-soft/src/main/java/softServe/academy/cinemasoft/model/Screening.java (hunk 2 above)
Comment (AtanasCh) [reply cinema-soft/src/main/java/softServe/academy/cinemasoft/model/Screening.java]: Done.

File: cinema-soft/src/main/java/softServe/academy/cinemasoft/model/Screening.java (hunk 3 above)
Comment (AtanasCh) [reply cinema-soft/src/main/java/softServe/academy/cinemasoft/model/Screening.java]: Fixed.

File: cinema-soft/src/main/java/softServe/academy/cinemasoft/controller/ScreeningController.java (hunk 1 above)
Comment (AtanasCh) [reply cinema-soft/src/main/java/softServe/academy/cinemasoft/controller/ScreeningController.java]: Fixed.

**Pull request merged**